07-routing-gossip.md

```diff
@@ -553,7 +553,7 @@ The final node:
         - Note: this will later allow the final node to rebuild the announcement
         for its peers.
 
-An endpoint node:
+A node:
```

Comment:
Yes; in fact, every occurance of `final node` or `endpoint node` in this BOLT are wrong...